commands/command_fetch.go

```diff
@@ -70,13 +89,74 @@ func fetchRefToChan(ref string, include, exclude []string) chan *lfs.WrappedPoin
 
 // Fetch all binaries for a given ref (that we don't have already)
 func fetchRef(ref string, include, exclude []string) {
-	pointers, err := lfs.ScanRefs(ref, "", nil)
+	pointers, err := pointersToFetchForRef(ref)
 	if err != nil {
 		Panic(err, "Could not scan for Git LFS files")
 	}
 	fetchPointers(pointers, include, exclude)
 }
 
+// Fetch all previous versions of objects from since to ref (not including final state at ref)
+// So this will fetch all the '-' sides of the diff from since to ref
+func fetchPreviousVersions(ref string, since time.Time, include, exclude []string) {
+	pointers, err := lfs.ScanPreviousVersions(ref, since)
+	if err != nil {
+		Panic(err, "Could not scan for Git LFS previous versions")
+	}
+	fetchPointers(pointers, include, exclude)
+}
+
+// Fetch recent objects based on config
+func fetchRecent(alreadyFetchedRefs []*git.Ref, include, exclude []string) {
+	fetchconf := lfs.Config.FetchPruneConfig()
+
+	if fetchconf.FetchRecentRefsDays == 0 && fetchconf.FetchRecentCommitsDays == 0 {
```

Comment:
I thought about using <= 0 (< 1 works except if we allow fractional days in future) but it just seemed less intuitive. As you say the config prevents any value < 0 anyway.